kube-derive/Cargo.toml

```diff
@@ -28,7 +28,7 @@ schema = []
 [dev-dependencies]
 serde = { version = "1.0.118", features = ["derive"] }
 serde_yaml = "0.8.17"
-kube = { path = "../kube", version = "<1.0.0, >=0.51.0" }
+kube-core = { path = "../kube-core" }
```

Comment:
Note to self: the version keys need to be in there to avoid cyclic dependencies in publishing, but maybe the first release needs to be special cased.